src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -14,11 +14,16 @@ import {
 } from '@material-ui/core';
 import { orderBy } from 'lodash';
 import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import { TableRow as AnalysisTableRow } from '../../../../context/analysisResultStateSlice';
 import { showPopup } from '../../../../context/tooltipStateSlice';
 import { Column } from '../../../../utils/analysis-utils';
+import { isLocalLanguageChosen, safeTranslate } from '../../../../i18n';
 
 function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
+  // only display local names if local language is selected, otherwise display english name
+  const { t, i18n } = useTranslation();
+  const filteredColumns = columns.filter(({ id }) => id !== 'localName');
```

Comment:
Another option would be to remove the `localName` column completely. And as the name column, load the english name or the local name depending on what i18n language is selected. This will also simplify the logic below.

Reply:
@tdlinh2712 what do you think?

Reply:
I tried to do it, however since `columns` are loaded from props, the value of name column are not reloaded when language is changed

Reply:
ok, good catch, then I guess we can leave it like that :)